contents/handbook/small-teams/growth/index.mdx

```diff
@@ -15,10 +15,19 @@ Maximize the number of people who get value out of PostHog, and help them realiz
 
 ## What the Growth team does
 
-- Sign up
-- Activation and reducing time to value
-- Billing and pricing changes
-- Product led growth, including messaging, emailing and notifications
+- Anything activation
+  - Activation to features or across features, but only for existing features.
+  - Onboarding flows (global and per feature)
+  - Sign up and authentication
+  - Helping product teams excel at activation on their own product
+- Product led growth
+  - Develop new products and strategies to increase top of funnel
+- Anything Billing
+  - Billing infrastructure
+  - Pricing (in collaboration with each product team)
+  - Revenue reporting
+  - Internal metrics reporting (company scorecard)
```

Comment:
This doesn't feel like it belongs below `Billing` since the scorecard should contain more than just revenue numbers.

```suggestion
- Internal metrics reporting (company scorecard)
```